calculator.js

```diff
@@ -1,26 +1,130 @@
+const Regione = {
+    ABRUZZO: "Abruzzo",
+    BASILICATA: "Basilicata",
+    CALABRIA: "Calabria",
+    CAMPANIA: "Campania",
+    EMILIA_ROMAGNA: "Emilia-Romagna",
+    FRIULI_VENEZIA_GIULIA: "Friuli Venezia Giulia",
+    LAZIO: "Lazio",
+    LIGURIA: "Liguria",
+    LOMBARDIA: "Lombardia",
+    MARCHE: "Marche",
+    MOLISE: "Molise",
+    PIEMONTE: "Piemonte",
+    PUGLIA: "Puglia",
+    SARDEGNA: "Sardegna",
+    SICILIA: "Sicilia",
+    TOSCANA: "Toscana",
+    TRENTINO_ALTO_ADIGE: "Trentino-Alto Adige",
+    UMBRIA: "Umbria",
+    VALLE_D_AOSTA: "Valle d'Aosta",
+    VENETO: "Veneto",
+}
+const findRegionByName = (regionString) => Object.keys(Regione).find(key => Regione[key] === regionString);
+
+const nomiRegioni = Object.keys(Regione).map(key => Regione[key]);
+
 const regioniToComuni = {
-    'Lazio': ['Roma', 'Latina', 'Rieti', 'Viterbo', 'Frosinone'],
-    'Sicilia': ['Catania', 'Palermo'],
-    'Campania': ['Caserta'],
-}
-var soglieIrpefToPercentualeMarginale = {
-    15_000 : 0.23, 
-    28_000 : 0.25,
-    50_000 : 0.35,
-    1_000_000_000: 0.43,
+    ABRUZZO: ["PLACEHOLDER"],
+    BASILICATA: ["PLACEHOLDER"],
+    CALABRIA: ["PLACEHOLDER"],
+    CAMPANIA: ['Caserta'],
+    EMILIA_ROMAGNA: ["PLACEHOLDER"],
+    FRIULI_VENEZIA_GIULIA: ["PLACEHOLDER"],
+    LAZIO: ['Roma', 'Latina', 'Rieti', 'Viterbo', 'Frosinone'],
+    LIGURIA: ["PLACEHOLDER"],
+    LOMBARDIA: ["PLACEHOLDER"],
+    MARCHE: ["PLACEHOLDER"],
+    MOLISE: ["PLACEHOLDER"],
+    PIEMONTE: ["PLACEHOLDER"],
+    PUGLIA: ["PLACEHOLDER"],
+    SARDEGNA: ["PLACEHOLDER"],
+    SICILIA: ['Catania', 'Palermo'],
+    TOSCANA: ["PLACEHOLDER"],
+    TRENTINO_ALTO_ADIGE: ["PLACEHOLDER"],
+    UMBRIA: ["PLACEHOLDER"],
+    VALLE_D_AOSTA: ["PLACEHOLDER"],
+    VENETO: ["PLACEHOLDER"],
 }
-var regioniToIrpef = {
-    'Lazio': 0.02,
-    'Sicilia': 0.0123,
-    'Abruzzo': 0.0173,
-    'Calabria': 0.0203,
-    'Campania': 0.0203,
-    'Sardegna': 0.0123,
-    // ...
-    // 'Lazio': {15_000: 0.0173, 28_000: 0.0273, 55_000: 0.0293, 75_000: 0.0323, 75_000: 3.33}
-    // could be map {threashold: percentage, ...}
+
+const regioniToScaglioniIrpef = {
+    ABRUZZO: {
+        1_000_000_000: 0.0173,
+    },
+    BASILICATA: {
+        55_000: 0.0123,
+        75_000: 0.0173,
+        1_000_000_000: 0.0233,
+    },
+    // "Regione"
```

Comment:
This is not a region but we should treat it as such.
I wonder if there is a cleaner way to do this than just adding this make this a `Region`.
@cpaoli